addons/website_quote/controllers/main.py

```diff
@@ -160,9 +145,9 @@ def add(self, option_id, order_id, token, **post):
             'discount': option.discount,
         }
 
-        line = request.registry.get('sale.order.line').create(request.cr, SUPERUSER_ID, vals, context=request.context)
-        request.registry.get('sale.order.line')._compute_tax_id(request.cr, SUPERUSER_ID, [line], context=request.context)
-        option_obj.write(request.cr, SUPERUSER_ID, [option.id], {'line_id': line}, context=request.context)
+        order_line = request.env['sale.order.line'].sudo().create(vals)
+        order_line._compute_tax_id()
+        option.line_id = order_line
```

Comment:
I prefer to keep write instead of direct assignation. It is easier to spot in the code when you actually modify a record.